doc/powered-by.md

```diff
@@ -67,4 +74,4 @@ Using APISIX as a gateway, it uses the functions of current limiting, speed limi
 Using APISIX as a Web ACL gateway to deal with backend OA systems traffic.
 
 ## HelloTalk, Inc.
-Using Apache APISIX as an API gateway to manage all API and SSL certificates in test\dev\CMS environment.
```

Comment:
do not remove `Apache`

Reply:
The name format of these user cases are not consistent. Or I could add Apache to all lines.